python/example/single_cell_allen.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env python3
+
+from collections import defaultdict
+import json
+import arbor as A
+import matplotlib.pyplot as plt
+import matplotlib.patches as patches
+import seaborn as sns
+import numpy as np
+import pandas as pd
+
+# Check arbor is loaded fine and print some config diagnostics
+print(A.config())
+
+# cable parameters
+class parameters:
```

Comment:
If you are worried about versions here, the pre-3.7 way of things is using a NamedTuple. Otherwise, a plain tuple does the trick.

Reply:
Namedtuples are not mutable, so not a great fit with the way they're preallocated in the dict. I prefer to use a class in order to have names for the fields to keep the code readable. I didn't notice a slowdown w.r.t the dataclass, if that is your concern.

Reply:
They do not need to be mutable
```
d['a'].foo = 42 # FAIL
d['b'] = (1, 2, 3) # FINE
```

Reply:
But it requires a less readable way of assignment, and that's the primary goal of this/any example. I'd like to keep the class, the function returns in about a second here, which is plenty fast. We can add the `@dataclass` decorator back when we drop 3.6.

Reply:
Sure, but that's not really the issue. You are using a `class` where you should be using an `object` of the `class`.
I'd argue that if you need mutable throw-away collections the Python way is to use either a List or a Dict.

Reply:
Afaik this is the only way (pre dataclass) to let you get/set the fields as member, which is all that I am after. Lists or Dicts will make the code less clear.